app/views/brexit_campaign/_featured_taxons.html.erb

```diff
@@ -1,12 +1,12 @@
-<% @taxons.in_groups_of(local_assigns.fetch(:display_full_width, true) ? 3 : 2, false).each_with_index do |taxons_group, row_index| %>
+<% @featured_links.in_groups_of(local_assigns.fetch(:display_full_width, true) ? 3 : 2, false).each_with_index do |links_group, row_index| %>
   <div class="grid-row" data-module="track-click">
-    <% taxons_group.each do |taxon| %>
+    <% links_group.each do |link| %>
       <div class="<%= local_assigns.fetch(:display_full_width, true) ? 'column-one-third' : 'column-half' %>">
         <div class="campaign__taxon <%= "campaign__taxon--variable-border" if row_index.eql?(0) %> ">
           <h3 class="campaign__taxon-title">
-            <%= link_to(taxon.title, taxon.finder_link, data: taxon.data_attributes, "aria-label": "#{t('campaign.see_more_in_taxon', topic: taxon.title.downcase)}") %>
+            <%= link_to(link.title, link.link, data: link.featured_data_attributes, "aria-label": "#{t('campaign.see_more_in_taxon', topic: link.title.downcase)}") %>
```

Comment:
If you have the index here, from the `each_with_index` above, could you provide this index value to `link.featured_data_attributes` and save yourself some work in the presenters? I suppose it's nice to keep the view lean, but since we already have the indexes to hand it would be simpler.

Reply:
Nice option.  I'll have a quick go at that, thanks